jjb/jjb-templates/jq-core-jobs.yaml

```diff
@@ -0,0 +1,84 @@
+---
+- job_boiler_plate: &job_boiler_plate
+    name: job-boiler-plate
+
+    project-type: freestyle
+    node: "{build-node}"
+
+    ######################
+    # Default parameters #
+    ######################
+
+    branch: main
+    submodule-recursive: true
+
+    #####################
+    # Job Configuration #
+    #####################
+
+    properties:
+      - lf-infra-properties:
+        project: "{project}"
+        build-days-to-keep: "{build-days-to-keep}"
+
+    wrappers:
+      - lf-infra-wrappers:
+        build-timeout: "{build-timeout}"
+        jenkins-ssh-credential: "{jenkins-ssh-credential}"
+
+    publishers:
+      - lf-infra-publish
+
+- verify_boiler_plate: &verify_boiler_plate
```

Comment:
This instance of the name, and the one directly below, should be `verify-boiler-plate` (the anchor should remain as underscores).